src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Compilation.targets

```diff
@@ -104,6 +111,12 @@ Copyright (c) .NET Foundation. All rights reserved.
       <UseSharedCompilation>true</UseSharedCompilation>
     </PropertyGroup>
 
+    <PropertyGroup>
+      <LangVersion Condition="'$(LangVersion)' == '' AND
+        (('$(TargetFrameworkIdentifier)' == '.NETCoreApp' AND '$(TargetFrameworkVersion)' == 'v3.0') OR
+         ('$(TargetFrameworkIdentifier)' == '.NETStandard' AND '$(TargetFrameworkVersion)' == 'v2.1'))">8.0</LangVersion>
```

Comment:
Forcing this to `8.0` until I can get confirmation from @dougbu or @anurse on if/how we can update the SDK in the AspNetCore-Tooling repo to get a newer Roslyn that understands `preview`

Reply:
@NTaylorMullen not sure what you're asking. Didn't your Versions.props change grab newer Roslyn bits?

Reply:
We're grabbing newer Roslyn bits to compile against for runtime scenarios but for build time scenarios we depend on the Roslyn that's shipped in the SDK. Currently the SDK Roslyn isn't new enough to understand the "preview" `LangVersion`

Reply:
Ah, got it. In the Arcade world, there's only one change needed but it's not automated (yet). Update the following bit of global.json with your favourite version:
``` json
  "sdk": {
    "version": "3.0.100-preview3-010280"
  }
```

Reply:
Also, AspNetCore is on preview4-X already so we should be in the clear for getting this in.